server/src/main/java/io/spine/server/integration/IntegrationBus.java

```diff
@@ -282,26 +291,27 @@ public void unregister(ExternalMessageDispatcher<?> dispatcher) {
      * request for external messages} for that purpose.
      *
      * @param  currentlyRequested
-     *         the set of message types that are now requested by this instance of
-     *         integration bus
+     *         the set of channel identifiers containing message types that are now requested
```

Comment:
`the set of channel IDs, which correspond to the currently requested types of messages.`